eng/docker-compose/.env.e2e

```diff
@@ -57,6 +57,18 @@ USE_API_SCHEMA_PATH=true
 API_SCHEMA_PATH=/app/ApiSchema
 ENABLE_MANAGEMENT_ENDPOINTS=true
 
+# JWT Authentication
+JWT_AUTHENTICATION_ENABLED=true
```

Comment:
A bunch of JWT configuration, mostly for `Microsoft.IdentityModel.Tokens.TokenValidationParameters`